public/ngapp/register/controllers.js

```diff
@@ -212,13 +228,13 @@ controllers.controller("RoutersCtrl",["$scope", '$rootScope', '$state', 'routerF
         var router = _.filter($scope.data, function(data){
```

Comment:
Declare variables at top of the function body like I did for `templateUrl`, `name` etc.